ccd-definition/ComplexTypes.json

```diff
@@ -1244,5 +1244,22 @@
     "FieldTypeParameter": "RespondentType",
     "ElementLabel": "Additional respondents",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42736,
+    "ID": "Orders",
+    "ListElementCode": "ordersAndDetails",
```

Comment:
Please rename to `orderType`

Reply:
Looks like further migration from excel to json. I’d be inclined to leave ‘Orders’ for now and tackle in a wider tech debt story. Would you agree?